games/devtest/mods/testtranslations/locale/translation_po.fr.po

```diff
@@ -0,0 +1,22 @@
+# Test Plural-Forms parsing
+msgid ""
+msgstr ""
+"Plural-Forms: nplurals=2; plural= (n-1+1)<=1 ? n||1?0:1 : 1?(!!2):2;"
```

Comment:
```suggestion
"Plural-Forms: nplurals=2; plural= (n-1+1)<=1 ? n||1?0:1 : 1?(!!2):2;"

msgctxt "testtranslations"
msgid "Test mod to test translations."
msgstr "Test mod to test translations (French!)"
```

Reply:
Looks like `GUIEngine::getContentTranslations` needs to be adjusted as well for this to work.

Reply:
Ah yes, file name needs to match - could add a new file for this `.po`. Also might be worth requiring or at least recommending that the filename matches the context

Reply:
It seems like I noticed two issues here:
* `msgctxt` appears to be required here even when it can be omitted.
* Adding a `msgctxt` field along with a comment (at the beginning of the entry) works but throws an error `Unable to parse po file: previous entry has no "msgid" field but is not empty`

I'll add the new translation file for now and address these issues later.

Reply:
I have added `testtranslations.fr.po` (https://github.com/minetest/minetest/pull/14726/commits/cb6f3c5ff498df8788ad1b93af17a96f19c4df8d) and fixed the two issues mentioned above (https://github.com/minetest/minetest/pull/14726/commits/b73ced9409ef3614a65c4bcfbdcc0ca2f1886124 and https://github.com/minetest/minetest/pull/14726/commits/1b943bc584d509631233a134af3ab174ac8eb716)